### PR TITLE
[Kotlin Migration ] TextCardExporterTest

### PR DESCRIPTION
**Pull request by Prince-kushwaha**
## Purpose / Description
Kotlin migration of TextCardExporterTest
- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/libanki/TextCardExporterTest.kt [hunk 1]
```diff
@@ -10,80 +10,82 @@
...
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.io.File
+import java.io.IOException
+import java.util.*
+
+@RunWith(AndroidJUnit4::class)
+class TextCardExporterTest : RobolectricTest() {
+    private var mCol: Collection? = null
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/libanki/TextCardExporterTest.kt:29]: `@KotlinCleanup("lateinit")`

**Review by david-allison — CHANGES_REQUESTED** [review]

**Review by david-allison — CHANGES_REQUESTED** [review]
```
> Task :AnkiDroid:ktlintTestSourceSetCheck FAILED
/home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/test/java/com/ichi2/libanki/TextCardExporterTest.kt:1:1 File must end with a newline (\n)
/home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/test/java/com/ichi2/libanki/TextCardExporterTest.kt:15:1 Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end
/home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/test/java/com/ichi2/libanki/TextCardExporterTest.kt:32:1 Needless blank line(s)

> Task :AnkiDroid:lintAnalyzePlayDebug

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':AnkiDroid:ktlintTestSourceSetCheck'.
> A failure occurred while executing org.jlleitschuh.gradle.ktlint.worker.ConsoleReportWorkAction
   > KtLint found code style violations. Please see the following reports:
     - /home/runner/work/Anki-Android/Anki-Android/AnkiDroid/build/reports/ktlint/ktlintTestSourceSetCheck/ktlintTestSourceSetCheck.txt
```

**Review by david-allison — APPROVED** [review]

**Review by mikehardy — APPROVED** [review]
thank you!

**Comment by mikehardy** [comment]
Hi there! APRIL OpenCollective Notice

Just a friendly notice that we try to process OpenCollective payments monthly - it's time for [April 2022 submissions](https://github.com/ankidroid/Anki-Android/pulls?q=is%3Apr+merged%3A2022-04-01..2022-04-30)

If you are interested in compensation for this work, the process with details is here:

https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process#how-to-get-paid

I only post one comment per person to avoid spamming you, regardless of the number of PRs merged, but this note applies to all PRs merged for in the month of March.

Please note that GSoC contributions are okay for this process. Our philosophy is that our users have donated to AnkiDroid for all contributions. The only PRs that will not go through the OpenCollective process are ones directly related to am accepted GSoC project from a selected participant, since those receive a stipend from GSoC itself.

Please understand that this means the monthly budget we have for payments might be stretched a bit and the cap on payments-per-person may be lower, but we try to make our process as fair and transparent as possible, we just need your understanding.

Thanks!

**Pull request merged**